pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -469,6 +469,11 @@ public List<InstanceConfig> getBrokerInstancesConfigsFor(String tableName) {
         TagNameUtils.getBrokerTagForTenant(brokerTenantName));
   }
 
+  public List<InstanceConfig> getAllBrokerInstanceConfigs() {
+    return HelixHelper.getInstanceConfigs(_helixZkManager).stream()
+        .filter(instance -> InstanceTypeUtils.isBroker(instance.getId())).collect(Collectors.toList());
+  }
```

Comment:
We can add API to get all broker instance ids which is much cheaper
```suggestion
  public List<String> getAllBrokerInstances() {
    return HelixHelper.getAllInstances(_helixAdmin, _clusterName).stream()
        .filter(instanceId -> InstanceTypeUtils.isBroker(instanceId)).collect(Collectors.toList());
  }
```